cmd/mattermost/commands/channel_test.go

```diff
@@ -30,24 +29,53 @@ func TestJoinChannel(t *testing.T) {
 }
 
 func TestRemoveChannel(t *testing.T) {
+
 	th := Setup(t).InitBasic()
 	defer th.TearDown()
 
 	channel := th.CreatePublicChannel()
 
-	th.CheckCommand(t, "channel", "add", th.BasicTeam.Name+":"+channel.Name, th.BasicUser2.Email)
-
-	// should fail because channel does not exist
```

Comment:
This case is missing after the refactor

Reply:
I've never seen this! Sorry about that @mgdelacroix I'll update this ASAP @jasonblais

Reply:
Done @mgdelacroix @jasonblais

Reply:
No worries! Thanks a lot for updating the PR :)